README.md

```diff
@@ -1,2 +1,53 @@
 # ocp-ci-docs
+
 Scenario onboarding documentation for OpenShift CI created by the CSPI team
+
+
+## Contributing
+
+### Best Practices
+
+- Ensure any new document is linked properly in the [`docs/SUMMARY.md`](docs/SUMMARY.md) file. This is how GitBook will present documents in the generated webpage.
+- All pull requests into the `main` branch of this repository should come from a branch in your [personal fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks) of the repository.
+- Do **NOT** push any changes to the `main` branch of this repository. Any changes to this branch must go through the [pull request process](#pull-requests).
+
+### Pull Requests
+
+- All pull requests must be reviewed by another member of the team before being merged.
+- Don't merge your own pull request. Ask someone that did a final review of your changes to merge your pull request when you are ready.
```

Comment:
```suggestion
- Do not merge your own pull request. Ask a reviewer who performed a final review of your changes to merge your pull request when you are ready.
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)